user-guide/Basic_Functionality/Alarms/Working_with_alarms/Advanced_analytics_features/Behavioral_anomaly_detection.md

```diff
@@ -66,15 +66,17 @@ To provide feedback:
 
    - Close the *Anomalies* or *Suggestion events* tab and reopen it. However, make sure not to use the *Reopen closed alarm lists* option. See [Adding a tab](xref:ChangingTheAlarmConsoleLayout#adding-a-tab).
 
+    This step might be necessary if you have recently upgraded from a version of DataMiner that did not include the Feedback column to one that does (DataMiner 10.4.11/10.5.0 or higher). Since Cube saves your open tabs along with their column layout, the new column will not automatically appear in tabs that were open before the upgrade.
```

Comment:
I think indentation isn't as intended here. This seems to be displayed as if outside the enumeration. I think if you add an extra space, it will should with the same indentation as the text above?

Reply:
Thanks for letting me know! This should now be fixed.